tests/util/checker/parser/testdata/data_test.go

```diff
@@ -0,0 +1,26 @@
+package testdata
+
+import (
+	"testing"
+
+	"istio.io/istio/tests/annotation"
+)
+
+func SetCount()       {}
+func Count(n int) int { return n }
+
+// nolint: testlinter
+func TestIsMarkedFlaky(t *testing.T) {
+	SetCount()
+	if Count(1) != 1 {
+		t.Error("expected 1")
+	}
+	if Count(2) != 2 {
+		t.Error("expected 2")
+	}
+	if Count(3) != 3 {
+		t.Error("expected 3")
+	}
+	t.Skip("https://github.com/istio/istio/issues/6041")
+	annotation.IsFlakyTest()
```

Comment:
I think we should recommend people putting this in the first line of the method. Can we check that too?